tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "pretty": true,
     "lib": ["es6", "es2015", "es2016", "es2017", "dom", "esnext.asynciterable"],
     "noUnusedLocals": true,
-    "strict": true
+    "strict": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Maybe enable `esModuleInterop` as well?

Reply:
Okay I will try

Reply:
It create more errors to fix, let's merge for now and enable it later :)